docs/source/workflow_inputs.rst

```diff
@@ -2473,6 +2474,8 @@ Each central air conditioner is entered as a ``/HPXML/Building/BuildingDetails/S
   .. [#] If CompressorType not provided, defaults to "single stage" if SEER <= 15, else "two stage" if SEER <= 21, else "variable speed".
   .. [#] The sum of all ``FractionCoolLoadServed`` (across all HVAC systems) must be less than or equal to 1.
   .. [#] If SEER2 provided, converted to SEER using ANSI/RESNET/ICC 301-2022 Addendum C, where SEER = SEER2 / 0.95 (assumed to be a split system).
+  .. [#] If neither EER nor EER2 provided, EER defaults to ... FIXME.
```

Comment:
I didn't bother to write out the equations yet since they still might change. We'll need to update this before everything gets merged down to the master branch.